src/DbDumper.php

```diff
@@ -159,6 +161,38 @@ public function setDumpBinaryPath(string $dumpBinaryPath)
         return $this;
     }
 
+    /**
+     * @return $this
+     */
+    public function enableCompression()
```

Comment:
👍 for ensuring backwards compatibility.

Mark this method as deprecated with the right docblock annotion. I'll remove the method when creating a new major version.

Reply:
Done.